Figgle.Generator.AcceptanceTests/FiggleSourceGeneratorAcceptanceTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System;
+using Xunit;
+
+namespace Figgle.Generator.AcceptanceTests;
+
+[GenerateFiggleText("HelloWorld", "My External Font", "Hello World!")]
```

Comment:
A possible extension to this would be to generate const strings for any external font names, so that if the file is removed or the build item removed, then the code wouldn't compile. Some open questions (like how to create a valid C# member name from an arbitrary file name) but thought I'd throw it out there. This PR is great as is.